types/tx.go

```diff
@@ -21,8 +21,7 @@ const (
 	DirectionSelf     Direction = "yourself"
 
 	TxTransfer              TransactionType = "transfer"
-	TxNativeTokenTransfer   TransactionType = "native_token_transfer"
```

Comment:
You need to keep native token transfer here - all toke transfers will use it

Reply:
just use `TxTokenTransfer`?

Reply:
`TxTokenTransfer` is `native_token_transfer` now

Reply:
👍

Reply:
after checking iOS code, I guess we'd better to keep both `TxNativeTokenTransfer` and `TxTokenTransfer` here? so it won't break its logic

Reply:
We can only return native transfers for all. For consistency

Reply:
Or wait until new V2 format is ready - that works too